src/components/Footer/Footer.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import Link from '../link/link';
-import Container from '../container/container';
+import Link from '../Link/Link';
+import Container from '../Container/Container';
```

Comment:
Ok still interested in thoughts on this ambiguity but as far as I can tell this isn't causing any issues.